### ISSUE TITLE
Builds failing at 1.1.14

### ISSUE THREAD

**Issue by paynoattn** [valor-software/ngx-bootstrap#1342]
All my teams builds are failing when v1.1.14 is set. This is fixed by updating to latest.

Build error: 
```
npm ERR! Error extracting /root/.npm/ng2-bootstrap/1.1.15/package.tgz archive: ENOENT: no such file or directory, open '/root/.npm/ng2-bootstrap/1.1.15/package.tgz'
```

**Comment by heresie** [comment]
Same here :
npm ERR! Error extracting /root/.npm/ng2-bootstrap/1.1.15/package.tgz archive: ENOENT: no such file or directory, open '/root/.npm/ng2-bootstrap/1.1.15/package.tgz'

**Comment by valorkin** [comment]
Most probably it is npm issue, just install latest

**Comment by sharok** [comment]
I have the same error. Npm is the latest.

**Comment by valorkin** [comment]
I mean npm install ng2-bootstrap@latest
I am still working on new docs but this should help http://valorkin.github.io/ng2-bootstrap/#/alerts

**Comment by nurandhawa** [comment]
Same here.

**Comment by vinaocruz** [comment]
+1

**Comment by valorkin** [comment]
node version
npm version
ng2-bootstrap version
error log

I will try to contact npm support

**Comment by perlauge** [comment]
As @valorkin  says it helps with a newer version - at least it helped me bumping the version from 1.1.15 to 1.1.16 in `package.json` for a different project.

**Comment by valorkin** [comment]
with ng v2.3 and v2.4 it is better to use 1.1.16-9 version +

**Comment by perrosen** [comment]
Same issue. Fixed by bumping 1.1.14 to 1.1.6 like @OmarMakled suggested. NPM was trying to install 1.1.15 so assume thats the actual version with the issue.
